src/app/material-module/material.module.ts

```diff
@@ -0,0 +1,88 @@
+import { NgModule } from '@angular/core';
```

Comment:
By importing everything and re-exporting it you kind of making a mess. I don't think having a material-module is a good practice. Please import just stuff you need from material directly into your module.